channels/drive/client/drive_file.c

```diff
@@ -426,7 +426,7 @@ BOOL drive_file_seek(DRIVE_FILE* file, UINT64 Offset)
 
 BOOL drive_file_read(DRIVE_FILE* file, BYTE* buffer, UINT32* Length)
 {
-	UINT32 read;
+	DWORD read;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW

Reply:
Yes, those are same 32bit unsinged integer but gcc shows warnings about using different "types" as ReadFile has DWORD in its function definition. These are just some warnings.